src/components/DonationForm/DonationForm.jsx

```diff
@@ -120,35 +278,103 @@ function DonationForm() {
               <Input {...register('email2')} />
               <FormErrorMessage>{errors.email2 && errors.email2.message}</FormErrorMessage>
             </FormControl>
-          </div>
-        </div>
+          </Box>
+        </Box>
 
-        <div className={styles['field-section']}>
+        {/* <Box className={styles['field-section']}>
           <h1 className={styles.title}>Furniture Submissions</h1>
-          {furnitureFields.map((furniture, index) => (
-            // eslint-disable-next-line react/jsx-key
-            <FurnitureField index={index} register={register} removeFurniture={removeFurniture} />
+          <FormControl isInvalid={errors && errors.Item}>
+            {itemsList.map((furniture, index) => {
+              if (index === 0) {
+                // eslint-disable-next-line react/jsx-key
+                return <FurnitureField index={index} register={register} />;
+              }
+              return (
+                // eslint-disable-next-line react/jsx-key
+                <FurnitureField index={index} register={register} removeFurniture={removeItem} />
+              );
+            })}
+          </FormControl>
+        </Box>
+
+        <Box className={styles['field-section']}>
+          <Button onClick={() => appendItem({})}>Add new furniture field</Button>
+        </Box> */}
+        <Select
+          placeholder="Select Furniture"
+          value={selectedFurnitureValue}
+          onChange={ev => onSelectFurniture(ev)}
+        >
+          {furnitureOptions.map((furnitureItem, i) => (
+            // eslint-disable-next-line react/no-array-index-key
```

Comment:
I'm pretty sure you don't need this eslint comment anymore since you already added the key

Reply:
If you do <option key={i}>{furnitureItem}</option> then eslint will give an error saying do not use an array index as a key, so the comment just disables that eslint error